data/src/main/java/com/linkedin/data/schema/resolver/FileDataSchemaResolver.java

```diff
@@ -184,6 +190,24 @@ public void setPaths(List<String> paths)
     return _paths;
   }
 
+  /**
+   * Return the current file path suffix for schemas location dir
+   *
+   * @return the current search paths.
+   */
+  public String getSchemasPathSuffix() {
+    return _schemasPathSuffix;
+  }
+
+  /**
+   * Sets the file path suffix for schemas location dir
+   *
+   * @param schemasPathSuffix path suffix.
+   */
+  public void setSchemasPathSuffix(String schemasPathSuffix) {
```

Comment:
I think we should call it "setSchemasDirectory", same for get method.

Reply:
updated to setSchemasDirPathSuffix